cmd/mal/mal.go

```diff
@@ -398,6 +398,9 @@ func main() {
 						return fmt.Errorf("scan: %w", err)
 					}
 
+					// Drop the applied filters
```

Comment:
We never display this via other rendering methods and it was causing a weird diff when testing the JSON files.